src/chapters/methods/studio.rst

```diff
@@ -0,0 +1,88 @@
+Studio: Methods
+====================
+
+The ``Reverse`` method flips the order of the elements within an array.
+However, ``Reverse`` does not affect the digits or characters within those
+elements.
+
+.. admonition:: Example
+
+   .. sourcecode:: csharp
+      :linenos:
+
+      string StringandArrayStudio01 = new string[] {"hello", "world", "123", "orange"};
+
+      Array.Reverse(arr);
+
+      foreach(string a in arr)
+      {
+         Console.WriteLine(a)
+      }
+
+   **Console Output**
+
+   ::
+      
+      orange
+      123
+      world
+      hello
+      
+
+What if we wanted the reversed array to be
+
+:: 
+ 
+   egnaro
+   321
+   dlrow
+   olleh
+
+Let's have some fun by creating a process that reverses BOTH the order of the
+entries in an array AND the order of characters within the individual elements.
+
+Remember that a method should perform only *one* task. 
+
+
+Warm-Up: Reverse a Number Array
+--------------------------------
+
+1. Define a method called ``ReverseNumbers`` that will sort the numbers within an array in descending order.
```

Comment:
here's where you can say: create a new repl ... yada yada ... don't forget to create a container ``MainClass`` and copy the ``Main`` method signature so that you can run the program and call your ``ReverseNumbers`` method